data/styles/clear/include/Icons.mapcss

```diff
@@ -2619,6 +2622,9 @@ area|z17-[craft=photographer]
 node|z17-[craft=plumber],
 area|z17-[craft=plumber]
 {icon-image:plumber-m.svg;}
+node|z17-[craft=sawmill],
+area|z17-[craft=sawmill]
+{icon-image:sawmill-m.svg;}
```

Comment:
This changes should be in first commit with icons, not in "Regenerated".

Reply:
You're right. Fixed!